doc/lightning-ping.7.md

```diff
@@ -11,7 +11,7 @@ DESCRIPTION
 
 The **ping** command checks if the node with *id* is ready to talk. It accepts the following parameters:
 
-- *id*: A string that represents the node id;
+- *id*: Hex-encoded pubkey of the target node;
```

Comment:
```suggestion
- *id*: A string, hex-encoded pubkey of the target node;
```

Reply:
How can a "hex-encoded something" not be a string?

Reply:
432421 ?